screens/BooksScreen.js

```diff
@@ -21,28 +21,69 @@ export default function BooksScreen() {
     getBooks();
   }, []);
 
-  const authorName = data.map((e) => {
-    e.authors.map((author) => {
-      console.log(author.name)
-      return author.name
-    })
-     });
+  //looking for ISBN
+  const availabilities =  data.map((book) => {
+    return !!book && book.availability
+  });
+
+  const isbnArray = availabilities.map((availability)=>{
+    console.log("availability.isbn:", availability !== undefined ? availability.isbn : '')
+    return availability !== undefined ? availability.isbn : ''
+  })
+    console.log("isbnArray:", isbnArray)
```

Comment:
output: <img width="163" alt="Screenshot 2021-11-01 at 17 22 52" src="https://user-images.githubusercontent.com/43641321/139706349-d469fbf9-4fea-483f-9a13-0a8d9f479b8c.png">
isbnArray is indeed an array of isbn different values